### ISSUE TITLE
colab notebook - more examples working from video

### ISSUE THREAD

**Issue by shawngraham** [distant-viewing/dvt#31]
Very much appreciate the colab notebook, as I find the documentation a bit hard to parse, given my low skill level. With regard to working with video it'd be good to have code snippets demonstrating the other annotators on video as in examples 9 - 12 using images.

**Comment by statsmaths** [comment]
Got it. Yes I will work on adding that (can also show it works to work with only a subset of the frame) as soon as possible.

And I agree about the documentation. Do you have any other Python projects whose documentation you find easier to parse? I know Python fairly well, but have always found the documentation tools available to be noticably worse than the other languages I use (R, JavaScript, C++, etc.).

**Comment by shawngraham** [comment]
I'm afraid that I always have struggles with documentation in all languages, ie, when things are broken down into individual bits and pieces. I learn best when I have a working example in front of me that I can follow along, parse, and then think 'ok, my situation is similar, so I ought to modify this at line ...' . I like R vignettes that give me some minimal examples (though of course, I'd always like more...)
